src/components/Menu/DropdownMenu.js

```diff
@@ -23,7 +23,7 @@ export default class DropdownMenu extends React.Component {
 
     return (
       <>
-        <a href="#" onClick={this.onMenuClicked}>
+        <a href="#target" onClick={this.onMenuClicked}>
```

Comment:
not sure I understand why `target`?

Reply:
ah I see, this is in a few other places - I don't think we need this, but I think we can clean all the instances at the same time. 👍